Fix grammatical error in learn-phragmen.md (#6066)
w3f/polkadot-wiki: docs/learn/learn-phragmen.md

The subset should be a minimum size of one (i.e., one cannot vote for no candidates) and a maximum
size of one less than the number of candidates (i.e., one cannot vote for all candidates). Users are

> allowed to vote for all or no candidates, but this will not have an effect on the final result, so
> votes of this nature are meaningless.

Reviewer: ```suggestion
allowed to vote for all or no candidates, but this will not affect the final result, rendering votes of this nature meaningless.
```
